packages/block-editor/src/components/block-lock/style.scss

```diff
@@ -12,13 +12,6 @@
 	margin: 0;
 }
 
-.block-editor-block-lock-modal__options-title {
```

Comment:
We shouldn't remove this css from block-editor package. I'll revert that and will remove the wrong class I had added in the checkbox.